### PR TITLE
Added support for passing the ssl_verify_peer flag to the AWS SDK

### PR DESCRIPTION
**Pull request by syedriko**
This is useful for testing against the https://github.com/spulec/moto AWS mock server, which by default runs with a self-signed certificate.

### CODE REVIEW

**Comment by jcantrill** [comment]
cc @cosmo0920

File: lib/fluent/plugin/out_cloudwatch_logs.rb [hunk 1]
```diff
@@ -120,6 +121,7 @@ def start
       options[:log_level] = :debug if log
       options[:region] = @region if @region
       options[:endpoint] = @endpoint if @endpoint
+      options[:ssl_verify_peer] = @ssl_verify_peer if @ssl_verify_peer
```
Comment (ashie) [inline lib/fluent/plugin/out_cloudwatch_logs.rb:124]: According to https://docs.aws.amazon.com/sdk-for-ruby/v3/api/Aws/CloudWatchLogs/Client.html it's `true` by default.
Although it seems that setting `false` is needed for your usage

> This is useful for testing against the https://github.com/spulec/moto AWS mock server, which by default runs with a self-signed certificate.

this code never set `false`.
Is it really the code that you expected?

Comment (syedriko) [reply lib/fluent/plugin/out_cloudwatch_logs.rb:124]: With this PR I meant to provide a conduit for setting ssl_very_peer in fluentd configuration. That's where I set it to false.

Comment (syedriko) [reply lib/fluent/plugin/out_cloudwatch_logs.rb]: I see what you mean now, my bad, fixed.

**Review by ashie — CHANGES_REQUESTED** [review]
Could you add an explanation about this option to README.md?

**Comment by syedriko** [comment]
> Could you add an explanation about this option to README.md?

With pleasure. I also added the new flag to in_cloudwatch_log for symmetry.

**Review by ashie — APPROVED** [review]

**Pull request merged**